projects/eslint-plugin-experience/README.md

```diff
@@ -8,3 +8,5 @@ module.exports = {
   extends: ['plugin:@taiga-ui/experience/all'],
 };
 ```
+
+Part of Taiga UI family.
```

Comment:
revert this file: remove "Part of Taiga UI family." and empty line before this text